app/global-translations/locale-en.json

```
@@ -213,6 +230,9 @@
   "label.anchor.editpermissions": "Edit Permissions",
   "#Inputs": "..",
   "label.input.name": "Name",
+  "label.input.unittype": "Unit Type",
+	"label.input.pcttobase": "Percentage To Base",
```

```suggestion
"label.input.pcttobase": "Percentage To Base",
```

You can directly apply these suggestions.

---

When I squash and re-push these changes won't get applied properly.  In my local branch these things are fixed